dotnet/src/SemanticKernel.Abstractions/Data/RecordOptions/DeleteRecordOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Data;
+
+/// <summary>
+/// Optional options when calling <see cref="IVectorStoreRecordCollection{TKey, TDataModel}.DeleteAsync"/>.
```

Comment:
Do we need to say "Optional options" or is just "Options" enough?

E.g. "Options that can be passed when calling ..."

Add a remark saying this is for future extensibility of the API as it currently doesn't include anything.

Reply:
Removed optional, it makes more sense where this is used.  Added the future extensibility remark.